NineChronicles.DataProvider/Store/MySqlStore.cs

```diff
@@ -104,5 +119,25 @@ private void Insert(string tableName, IReadOnlyDictionary<string, object> data)
                 }
             }
         }
+
+        private void Delete(string tableName, IReadOnlyDictionary<string, object> data)
+        {
+            using QueryFactory db = OpenDb();
+            var query = db.Query(tableName);
+            try
+            {
+                foreach (KeyValuePair<string, object> pair in data)
+                {
+                    query.Where(pair.Key, pair.Value);
```

Comment:
Is it okay without re-assigning?
```suggestion
                    query = query.Where(pair.Key, pair.Value);
```

Reply:
I was thinking the same thing but it either way works when I tested it. I'll apply your suggestion to be safe 👍